crates/cli/src/file_utils.rs

```diff
@@ -16,6 +16,7 @@ use kernel::model::violation::Violation;
 static FILE_EXTENSIONS_PER_LANGUAGE_LIST: &[(Language, &[&str])] = &[
     (Language::Csharp, &["cs"]),
     (Language::Dockerfile, &["docker", "dockerfile"]),
+    (Language::Elixir, &["ex"]),
```

Comment:
Add .exs as well, which is the scripting file extension - https://hexdocs.pm/elixir/modules-and-functions.html#scripting-mode

```suggestion
    (Language::Elixir, &["ex", "exs"]),
```